app/src/main/java/ApiHandler.java

```diff
@@ -0,0 +1,10 @@
+import com.kanshu.kanshu.KanshuApi;
```

Comment:
I'm not sure why you seem to have this file in both the com.kanshu.kanshu package and outside it?